CHANGELOG.md

```diff
@@ -1,11 +1,23 @@
 # App Center SDK for .NET Change Log
 
-## Version 1.13.1 (Not yet released)
+## Version 1.14.0 
```

Comment:
1.13.1 was good, we bump minor number if there is a least 1 feature, which is not the case here. Also both native SDKs that xamarin depends on were bug fixes only as well. Please revert to 1.13.0.